config.json

```diff
@@ -306,9 +306,9 @@
           }
         },
         "buildArch": {
```

Comment:
nit: but at some point we should consider changing this alias to just "arch" given we are using it for building and testing.